### PR TITLE
Transaction with withdrawal account

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

If user is warned about remaining ckBTC funds that haven't been converted to BTC then user should be able to open the transaction modal to execute again the transaction.

This PR link the opening of the modal with the call to action and set the options for such use case.

# Changes

- open transaction modal
- set few readonly and display options of the transaction modal if withdrawal account is provided

# Screenshots

Capture d’écran 2023-05-05 à 08 23 38
Capture d’écran 2023-05-05 à 08 24 03
Capture d’écran 2023-05-05 à 08 24 06

### CODE REVIEW

File: frontend/src/lib/components/accounts/CkBTCWithdrawalAccount.svelte [hunk 1]
```diff
@@ -93,6 +98,35 @@
         setTimeout(() => (visibility = "hidden"), 200);
       }, 1800);
     })();
+
+  const openSend = () => {
+    const canisters = nonNullish($selectedCkBTCUniverseIdStore)
+      ? CKBTC_ADDITIONAL_CANISTERS[$selectedCkBTCUniverseIdStore.toText()]
+      : undefined;
+
+    // Button is not displayed if no universe
```
Comment (lmuntaner) [inline frontend/src/lib/components/accounts/CkBTCWithdrawalAccount.svelte:107]: Is it displayed if no canisters?

File: frontend/src/tests/lib/components/accounts/CkBTCWithdrawalAccount.spec.ts [hunk 2]
```diff
@@ -242,4 +248,52 @@ describe("CkBTCWithdrawalAccount", () => {
...
+          accounts: [mockCkBTCMainAccount],
+          certified: true,
+        },
+        universeId: CKTESTBTC_UNIVERSE_CANISTER_ID,
+      });
+
+      tokensStore.setTokens(mockTokens);
+    });
+
+    it.only("should open send modal", async () => {
```
Comment (lmuntaner) [inline frontend/src/tests/lib/components/accounts/CkBTCWithdrawalAccount.spec.ts:277]: Remove the `only` I guess?

**Review by lmuntaner — APPROVED** [review]
A couple of questions but looks good to me, already approving!

File: frontend/src/lib/components/accounts/CkBTCWithdrawalAccount.svelte (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/components/accounts/CkBTCWithdrawalAccount.svelte:107]: I removed the comment, does not bring much.

There is no canisters definition without universe.

```
export const CKBTC_ADDITIONAL_CANISTERS: Record<
  UniverseCanisterIdText,
  CkBTCAdditionalCanisters
> = {
  [CKBTC_UNIVERSE_CANISTER_ID.toText()]: {
    indexCanisterId: CKBTC_INDEX_CANISTER_ID,
    minterCanisterId: CKBTC_MINTER_CANISTER_ID,
  },
  [CKTESTBTC_UNIVERSE_CANISTER_ID.toText()]: {
    indexCanisterId: CKTESTBTC_INDEX_CANISTER_ID,
    minterCanisterId: CKTESTBTC_MINTER_CANISTER_ID,
  },
};
```

File: frontend/src/tests/lib/components/accounts/CkBTCWithdrawalAccount.spec.ts (hunk 2 above)
Comment (peterpeterparker) [reply frontend/src/tests/lib/components/accounts/CkBTCWithdrawalAccount.spec.ts]: Thanks

File: frontend/src/lib/components/accounts/CkBTCWithdrawalAccount.svelte (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/components/accounts/CkBTCWithdrawalAccount.svelte]: Ok, thanks for the explanation!

**Pull request merged**